src/haxelib/server/Repo.hx

```diff
@@ -80,6 +80,15 @@ class Repo implements SiteApi {
 		};
 	}
 
+	public function getLatestVersion( project : String ) : SemVer {
+		var p = Project.manager.select($name == project);
+		if( p == null )
+			throw "No such Project : "+project;
+
+		var v = Version.manager.unsafeObjects('SELECT * FROM Version WHERE project == ${p.id} ORDER BY major DESC, minor DESC, patch DESC, previewNum DESC LIMIT 1', false);
```

Comment:
I think this is not correct, since it doesn't account `preview` field that should be also considered for sorting. Not sure if it can be done within a simple SQL query.